tests/keras/engine/test_training.py

```diff
@@ -594,6 +776,8 @@ def test_model_with_partial_loss():
     loss = {'dense_2': 'mse'}
     model.compile(optimizer, loss, metrics={'dense_1': 'mae'})
 
+    out = call_model_methods(model, input_a_np, output_a_np)
```

Comment:
What is the point to assigning to out so many times -- we don't see it anywhere else